Makefile

```diff
@@ -7,7 +7,8 @@ install: fmt
 	go install ./...
 
 clean:
-	rm -rf hostdir release whitepaper.aux whitepaper.log whitepaper.pdf sia.wallet sia/test.wallet sia/hostdir* sia/renterDownload
+	rm -rf hostdir release whitepaper.aux whitepaper.log whitepaper.pdf         \
+	sia.wallet sia/test.wallet sia/hostdir* sia/renterDownload
```

Comment:
subsequent lines after `\` should be indented (unless that breaks the Makefile, which isn't out of the question)